test-dev/main.c

```diff
@@ -1,6 +1,6 @@
 #ifdef WIN32
 #include <process.h> /* _spawnl, _P_WAIT */
-#elif !defined(__riscos__)
+#elif !defined(__riscos__) && !defined(NO_FORK_TEST)
```

Comment:
What's this `NO_FORK_TEST`?

Reply:
This macro I added to give the ability to enforce the no-fork mode, I used that to run one of the test cases directly, I.e. `libxmp-tests-dev 125` to debug it. That how I checked the reason for #414, right now the CMake flag was added that will appear once you enable the build of unit tests

Reply:
https://github.com/libxmp/libxmp/blob/dd9ad566c6944dbca78a240257b2aa49702b22d1/test-dev/CMakeLists.txt#L49-L52

Reply:
OK